src/components/PageDescription/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Image, Heading, Text } from 'grommet';
+
+const PageDescription = ({ image, title, description1, description2, children }) => {
+  return (
+    <Box
+    flex
+    overflow="auto"
+    gap="medium"
+    pad="large"
+    wrap
+  >
+    <Box direction="row-responsive" align="center" gap="large" pad={{vertical:"large"}}>
+        <Box pad={{horizontal:"large"}}>
+            <Image  src={image} />
+        </Box>
+        <Box
+        align="start"
+        direction="column"
+        >
+            <Heading margin="none">{title}</Heading>
+            <Text>
```

Comment:
The point of having the 'children' was to put {children} here rather than having odd description1/2 props.